_data/projects/BotKube.yml

```
@@ -0,0 +1,15 @@
+name: BotKube
+desc: App that helps you monitor your Kubernetes cluster, debug critical deployments & gives recommendations for standard practices
+site: https://www.botkube.io/
+tags:
+- kubernetes
+- kubernetes-monitoring
+- kubernetes-controller
+- bot
+- chatops
+- kubernetes-bot
+- slack-bot
+- mattermost-bot
+upforgrabs:
+  name: help wanted
+  link: https://github.com/infracloudio/botkube/issues?q=is%3Aissue+label%3A%22help+wanted%22+is%3Aopen
```

There's a shorter URL that will get you what you need here which is easier to read:

```suggestion
  link: https://github.com/infracloudio/botkube/labels/help%20wanted
```

---

@shiftkey Done!